docassemble/AssemblyLine/data/sources/languages.yml

```
@@ -0,0 +1,141 @@
+en: 
+  name: English
+  native_name: English
+es:
+  name: Spanish
+  native_name: Español
+zh: 
+  name: Chinese
+  native_name: 中文
+fr: 
+  name: French
+  native_name: Français
+de: 
+  name: German
+  native_name: Deutsch
+ru: 
+  name: Russian
+  native_name: Русский
+hi: 
+  name: Hindi
+  native_name: हिन्दी
+ar: 
+  name: Arabic
+  native_name: العربية
+ja: 
+  name: Japanese
+  native_name: 日本語
+ko: 
+  name: Korean
+  native_name: 한국어
+vi: 
+  name: Vietnamese
+  native_name: Tiếng Việt
+it: 
+  name: Italian
+  native_name: Italiano
+pt: 
+  name: Portuguese
+  native_name: Português
+tl: 
+  name: Tagalog
+  native_name: Wikang Tagalog
+pl: 
+  name: Polish
+  native_name: Polski
+nl: 
+  name: Dutch
+  native_name: Nederlands
+el: 
+  name: Greek
+  native_name: Ελληνικά
+ht: 
+  name: Haitian Creole
+  native_name: Kreyòl ayisyen
+zh-Hans: 
+  name: Mandarin (Simplified)
+  native_name: 普通话
+zh-Hant: 
+  name: Cantonese (Traditional)
+  native_name: 粵語
+kea: 
+  name: Cape Verdean Creole
+  native_name: Kriolu
+bn: 
+  name: Bengali
+  native_name: বাংলা
+so: 
+  name: Somali
+  native_name: Soomaali
+hy: 
+  name: Armenian
+  native_name: Հայերեն
+hmn: 
+  name: Hmong
+  native_name: Hmoob
+km: 
+  name: Khmer
+  native_name: ភាសាខ្មែរ
+om: 
+  name: Oromo
+  native_name: Afaan Oromoo
+am: 
+  name: Amharic
+  native_name: አማርኛ
+ne: 
+  name: Nepali
+  native_name: नेपाली
+lo: 
+  name: Lao
+  native_name: ພາສາລາວ
+my: 
+  name: Burmese
+  native_name: ဗမာစာ
+ps: 
+  name: Pashto
+  native_name: پښتو
+ku: 
+  name: Kurdish
+  native_name: Kurdî
+ta: 
+  name: Tamil
+  native_name: தமிழ்
+az: 
+  name: Azerbaijani
+  native_name: Azərbaycan
+yo: 
+  name: Yoruba
+  native_name: Yorùbá
+te: 
+  name: Telugu
+  native_name: తెలుగు
+uz: 
+  name: Uzbek
+  native_name: Oʻzbek
+pa: 
+  name: Punjabi
+  native_name: ਪੰਜਾਬੀ
+fa: 
+  name: Persian (Farsi)
```

```suggestion
  name: Farsi (Persian)
```

Fairly minor, and up to debate, but Farsi is more specific than Persian, and might be more familiar to our audience of the Iranian diaspora

---

This is fine, but note we never show the English name of the language